docs/source/support/compatibility.rst

```diff
@@ -0,0 +1,53 @@
+
+Specifically, if a DAML Application is built against Ledger API version X.Y.Z and a Participant Node exposes Ledger API version X.Y2.Z2, the application is guaranteed to work as long as Y2.Z2 >= Y.Z.
+
+Currently, the Ledger API version is the same as the version of the DAML Drivers, as everything gets released together as per :doc:`releases`, and there has been no need for the versions to diverge yet. This will likely change at the latest when one part of the ecosystem moves to version 2.X.
+
+As a concrete example, DAML for Postgres 1.4.0 exposes Ledger API version 1.4.0 and DAML for VMware Blockchain 1.0 exposes Ledger API version 1.6.0. So any application that runs on DAML for Postgres 1.4.0 will also run on DAML for VMware Blockchain 1.0, thus demonstrating both portability and stability.
+
+Ledger API Support
+------------------
+
+Major Ledger API versions behave like stable features in :doc:`status-definitions`. They are supported from the time they are first released as "stable" to the point where they are removed from DAML Drivers and SDK following a 12 month deprecation cycle. The earliest point a major Ledger API version can be deprecated is with the release of the next major version. The earliest it can be removed, is 12 months later with a major version release of DAML Drivers and Participant Nodes.
+
+Other than for hotfix releases, new releases of the DAML Drivers and Participant Nodes will only support the latest minor/patch version of each major Ledger API version.
+
+Network Upgradeability
+----------------------
+
+Upgrades from one minor version of a stable DAML Network or Participant nodes are data preserving and have Ledger API backward compatibility as major Ledger API versions may only be removed in major versions of DAML Drivers or Participant Nodes. As an example, from an application standpoint, the only effect of upgrading DAML for Postgres 1.4.0 to DAML for Postgres 1.6.0 is an uptick in the Ledger API version. There may be significant changes to components or database schemas, but these are not public APIs, and the migrations happen automatically. 
+
+SDK, Runtime Component, and Library Compatibility
+-------------------------------------------------
+
+As long as a major Ledger API version is supported, there will be supported versions of SDK, Runtime Components, and Libraries able to target those versions. This has the obvious caveat that new features may not be available with old Ledger API versions.
```

Comment:
No, that's not correct. Assume someone is on a Ledger exposing Ledger API Version 1.4, and they encounter a bug in the SDK forcing them to upgrade to SDK or Runtime Components 1.6. They still need to be able to maintain their app against Ledger API Version 1.4.

Reply:
So SDK 1.6 needs to support Ledger API 1.4, 1.5 and 1.6? Is there such a thing as a bugfix release for an API? (there obviously is for an implementation of that API but that’s a separate topic). I’d like to avoid having to support old bugs in the API that we fixed in newer patch versions but maybe that’s just not a topic that is meaningful for an API and only make sense in the context of an implementation.

Reply:
> SDK 1.6 needs to support Ledger API 1.4, 1.5 and 1.6
Yes, in the sense that you need to be able to test your application against those (ie something akin to `platform-version` called `ledger-api-version`.

Reply:
> I’d like to avoid having to support old bugs in the API that we fixed in newer patch versions but maybe that’s just not a topic that is meaningful for an API and only make sense in the context of an implementation

Fully agree as per your Fabric example, and I also can't see what an API patch release would be.

Reply:
Alright, then I would suggest to ignore my comments on bugfixes until we can come up with a meaningful example where this matters for an API.